.github/workflows/test.yml

```diff
@@ -34,30 +34,32 @@ jobs:
       - name: Test
         run: python -m pytest -v -m "not ctest"
 
-  build-openassetio:
-    name: Build OpenAssetIO
-    runs-on: ubuntu-latest
-    container:
-      image: ghcr.io/openassetio/openassetio-build
-    steps:
-      - uses: actions/checkout@v3
-      - name: Build
-        uses: ./.github/build_openassetio
-
   ctest:
-    name: C++ tests
-    runs-on: ubuntu-latest
-    needs: build-openassetio
-    container:
-      image: aswf/ci-vfxall:2022-clang14.3
+    name: C++ tests ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['windows-latest', 'ubuntu-latest', 'macos-latest']
     steps:
       - uses: actions/checkout@v3
 
-      - name: Get OpenAssetIO
-        uses: actions/download-artifact@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+
+      - name: Download Latest OpenAssetIO Artifact
+        uses: dawidd6/action-download-artifact@v2
         with:
-          name: OpenAssetIO Build
-          path: ${{ github.workspace }}/openassetio
+          name: openassetio-${{ runner.os }}
+          repo: elliotcmorris/openassetio
```

Comment:
[fix] I commited the change to main on my fork for testing. Obviously this must change to `openassetio/openassetio` before merge.

Reply:
https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/56/commits/e5f479c9f377db586d300894997096485715c359